static/bidder-info/adport.yaml

```diff
@@ -2,8 +2,8 @@ endpoint: "https://adport.pbsadverxo.com/auction?id={{.AdUnit}}&auth={{.TokenID}
 aliasOf: "adverxo"
 userSync:
   iframe:
-    url: https://adport.pbsadverxo.com/usync?type=iframe&gdpr={{.GDPR}}&consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&redirect={{.RedirectURL}}
+    url: https://cittamatra.com/usync?type=iframe&gdpr={{.GDPR}}&consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&redirect={{.RedirectURL}}
     userMacro: $UID
   redirect:
-    url: https://adport.pbsadverxo.com/usync?type=image&gdpr={{.GDPR}}&consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&redirect={{.RedirectURL}}
+    url: https://cittamatra.com/usync?type=image&gdpr={{.GDPR}}&consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&redirect={{.RedirectURL}}
```

Comment:
Verified bidsmind and adport iframe urls hit setuid endpoint with uid in cookie
<img width="1195" alt="Screenshot 2025-06-05 at 3 24 25 PM" src="https://github.com/user-attachments/assets/1eaaff6a-98e1-4b08-9735-d99e771c6831" />